state/payloads.go

```diff
@@ -100,6 +101,7 @@ func (UnitPayloads) LookUp(name, rawID string) (string, error) {
 // is already in the DB then it is replaced.
 func (up UnitPayloads) Track(pl payload.Payload) error {
 
+	// anastasiamac: not sure why is this a problem if the unit is known here?...
```

Comment:
it's not a problem for users because we're second guessing the nonsense from the payload/context code. did you not balk when you saw `Unit:   "a-application/0", // TODO(ericsnow) eliminate this!` in production code?